### DISCUSSION TITLE
Prompt Not Visible On Startup (Arch Linux)

### DISCUSSION THREAD

**Discussion by 0x4a61636f62** [ghostty-org/ghostty#3830] [Feature Requests, Ideas] [answered]
For me, the prompt is not visible on startup of **Gnome and Hyprland**. However, when I start typing, the prompt appears.

![prompt-not-visible-on-startup](https://github.com/user-attachments/assets/770f1dc3-a37a-404c-89c5-e7876c8ce9fd)

I am on 1.0.0-3 and installed the package listed on  **`extra/ghostty`** on Arch.

Has anyone else had the same issue?

**Comment by tuananh** [comment, marked as answer]
possibly related? https://github.com/ghostty-org/ghostty/discussions/3476

**Reply by 0x4a61636f62** [reply]
Thank you @tuananh, referencing that issue.
